### ISSUE TITLE
Request: add v-model.lazy for components

### ISSUE THREAD

**Issue by pdanpdan** [vuejs/vue#6914]
### What problem does this feature solve?
v-model for components already supports .trim and .number.
Lazy modifier support for components would make it more uniform across the platform.

### What does the proposed API look like?
The only change (backwards compatible) would be to add a new property to model definition:
```
model: {
  prop: 'value',
  event: 'input',
  eventLazy: 'change'
}
```

**Comment by Mikilll94** [comment]
Is it going to be done?

**Comment by iec989** [comment]
I think a bunch of us are desperately waiting for this; mid-size forms get very slow as soon as you implement some basic VUE logic around the components, we really need this!

**Comment by Mikilll94** [comment]
yeah, exactly. I have an app made with Vuetify and on larger forms instead of using v-model I am typing
`:value="sth" @change="($event) => sth = $event.target.value"`

**Comment by wc-matteo** [comment]
I would love to have it as well!

**Comment by mrcnk** [comment]
Also custom rules for Vuelidate need sometimes `v-model.lazy`, so it'd be useful not to introduce workarounds for it 🙌

**Comment by NikodemBieniek** [comment]
This would be handy

**Comment by Tantawi** [comment]
Please add this!

**Comment by danielbonifacio** [comment]
Waiting for it! Would be a great enhance on productivity 😄

**Comment by lxr900** [comment]
This would keep components much cleaner especially when binding to vuex data. Being able to use the `.lazy` modifier would allow me to keep both `get` and `set` functions in the computed properties rather than having to get the data in computed and commit the mutation with a method. Please add!!!

**Comment by pavlovskyarto** [comment]
Please, add this asap :+1:

**Comment by mes01** [comment]
Pleeeeeeeeeeease

**Comment by Sjoerd82** [comment]
I have the same need, and also I don't understand how to do a workaround until then. Example? Mikilll94's example gives me an error: "Cannot read property 'value' of undefined"

**Comment by jjsty1e** [comment]
![image](https://user-images.githubusercontent.com/5469845/173546947-c5aaa100-9040-4f06-80b2-29f665c29854.png)
